### DISCUSSION TITLE
HOW TO USE GSAP IN NEXT JS V-14 SSR

### DISCUSSION THREAD

**Discussion by saravanansundar823** [vercel/next.js#60427] [Help]
### Summary

Hello Everyone, 
How to solve this problem,
![Screenshot 2024-01-09 181941](https://github.com/vercel/next.js/assets/114160338/5ef1a33a-7d77-4355-9ccd-b65b8591fd33)

Unhandled Runtime Error
Error: async/await is not yet supported in Client Components, only Server Components. This error is often caused by accidentally adding `'use client'` to a module that was originally written for the server.

**Comment by ananuness** [comment]
You can separate the `async function` in another file and call it in your Client component, as you can use Server components/functions in your client.
